src/lib/metrics.js

```diff
@@ -160,12 +166,15 @@ class MetricsMonitor {
     }
 
     registerPoolMetrics(pool, eventStore) {
-        eventStore.emit(DB_POOL_UPDATE, {
-            used: pool.numUsed(),
-            free: pool.numFree(),
-            pendingCreates: pool.numPendingCreates(),
-            pendingAcquires: pool.numPendingAcquires(),
-        });
+        try {
+            eventStore.emit(DB_POOL_UPDATE, {
```

Comment:
not related to this change, but I did not notice this before. `EventStore` should generally be used for all events that are actually stored. We are currently using the `EventBus` for more internal metrics types of events.

Reply:
(I did not notice when this was introduced)

Reply:
Yeah, I'll fix that

Reply:
Fixed